prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/TN/TN-receiver-transform.yml

```diff
@@ -0,0 +1,49 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  MSH: '/MSH'
+
+elements:
+  - name: tn-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: tn-sending-application-namespace-id
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ '%{MSH}-3-1']
+
+  - name: tn-sending-application-universal-id
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ '%{MSH}-3-2' ]
+
+  - name: tn-sending-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-3-3' ]
+
+  - name: tn-receiving-application-namespace-id
+    value: [ '"tdh"' ]
+    hl7Spec: [ '%{MSH}-5-1' ]
+
+  - name: tn-receiving-application-universal-id
+    value: [ '"2.16.840.1.113883.3.773.1.1.3"' ]
+    hl7Spec: [ '%{MSH}-5-2' ]
+
+  - name: tn-receiving-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-5-3' ]
+
+  - name: tn-receiving-facility-namespace-id
+    value: [ '"TDH"' ]
+    hl7Spec: [ '%{MSH}-6-1' ]
+
+  - name: tn-receiving-facility-universal-id
+    value: [ '"2.16.840.1.113883.3.773"' ]
+    hl7Spec: [ '%{MSH}-5-2' ]
```

Comment:
This should be '%{MSH}-6-2'

Reply:
That is not where it was falling in the message. Maybe I need to change the name?

Reply:
Ah, got you . Thank you!